docs/concepts/what-is-a-codec-server.md

```diff
@@ -0,0 +1,28 @@
+---
+id: what-is-a-codec-server
+title: What is a Codec Server?
+sidebar_label: Codec Server
+description: Explanation and implementation of a remote encryption/decryption server.
+---
+
+A Codec Server is an optional feature used to encrypt and decrypt Payloads for the WebUI.
+
+![](/img/remote-codec-server-problem.svg)
+
+Each Client has a built-in [Data Converter](/docs/concepts/what-is-a-data-converter). However, the pre-built tctl and WebUI binaries won't be able to decrypt the serialized Payloads.
+
+![](/img/remote-codec-server-solution.svg)
+
+With a Codec Server, the Client's Data Converter can send an HTTP API request to the codec to decrypt a Payload.
+This decrypted data is passed back to the WebUI.
+
+Decrypted data can also be passed back through the Codec Server for encryption. Then, when passed through the Client's Data Converter, it can be deserialized before being sent back to the Temporal Cluster.
+
+![](/img/remote-codec-server-diagram.svg)
+
+## Configuration
```

Comment:
I don't think we need this header.

What we need is an additional "how to develop a Codec Server" or "How to run a Codec Server" page -